18 feature create tabtsx component

edwaddle opened the pull request:
Unsure if this how you want it to be formatted

edwaddle committed:
Created Tabs component

Currently testing it on the error screen, but will get rid of it when I finish the actual component. Using 'customText' as a placeholder since I'm not sure how to use Props yet

edwaddle committed:
Got rid of comments and the placeholder I used for the props element

Not sure if this is formatted correctly

ahmadgaz commented on app/error.tsx:
This file is unrelated, make sure to pull dev and `git checkout <source-branch> -- <path/to/file>` to reset the file so that it isn't included in the PR.

ahmadgaz replied:
Make sure this file is unchanged in the last commit

edwaddle committed:
Added max width for tabs/ semi responsive, turned tabs into radio buttons, Added line/ opacity difference when tab is selected

Currently stuck on how to make it so that only one radiobutton can be checked at a time/ how I can default one tab to being open

edwaddle committed:
Minor tweaks

I don't actually rememebr what I changed, I'm pretty sure it was something to do with the 'peer-checked' styling

edwaddle committed:
Finalized tab component

edwaddle committed:
added tabs to index.ts in atoms

ahmadgaz reviewed (changes requested):
awesome, just make sure to see the comment i made on error.tsx.

edwaddle committed:
Reverted error.tsx back to its original state

edwaddle committed:
Merge branch 'main' of https://github.com/mlatsjsu/mlatsjsu-website into 18-feature-create-tabtsx-component

ahmadgaz reviewed (changes requested):
remove error.tsx fiile

edwaddle commented:
Not sure why it's not showing that I committed, but only 3 of the files are changed now
